src/esp/sensor/Sensor.h

```diff
@@ -37,14 +37,20 @@ enum class ObservationSpaceType {
   TEXT = 2,
 };
 
+enum class SensorSubtype {
+  PINHOLE = 0,
+  ORTHOGRAPHIC = 1,
+};
+
 // Specifies the configuration parameters of a sensor
 struct SensorSpec {
   std::string uuid = "rgba_camera";
   SensorType sensorType = SensorType::COLOR;
-  std::string sensorSubtype = "pinhole";
```

Comment:
I think before we use string here is because it would be very convenient to set it from the python interface via python bindings.

Reply:
That's a great point!

Reply:
Enum is just as convenient.